generators/server/templates/quarkus/src/main/docker/Dockerfile.fast-jar.ejs

```diff
@@ -0,0 +1,56 @@
+####
+# This Dockerfile is used in order to build a container that runs the Quarkus application in JVM mode
+#
+# Before building the docker image run:
+#
+# mvn package
```

Comment:
If we will support gradle too we have to change this according to the choosen option